styles/components/_onboarding.scss

```diff
@@ -0,0 +1,96 @@
+.onboarding {
+  .modalContent {
+    .mainContent {
+      padding: $pad 0;      
+    }
+
+    .introScreen {
+      .mainContent {
+        height: 300px;
+
+        .btcIcon {
+          position: relative;
+          top: 2px;
+        }
+
+        .headline {
+          font-size: 3.6rem;
+        }
+
+        .btnGetStarted {
+          font-size: 1.8rem;
+          min-width: 275px;
+          height: 52px;
+          text-transform: uppercase;
+        }
+      }
+    }
+
+    .infoScreen {
+      form {
+        width: 375px;
+        margin: 0 auto;
+
+        .select2 {
+          // without this the select2 dropdowns exceed the width of the form on re-render
+          width: 375px !important;
```

Comment:
Initially it was the right width, but then if I clicked next and back, it would be the wrong with. But... I can't seem to reproduce that anymore. I updated the CSS.